CompanyAddress.md

```diff
@@ -0,0 +1,130 @@
+
+## Request Structure
+```json
+{
+	"latitude": 59.439675,
+	"longitude": 24.746831,
+	"company_name": "Texas Honky Tonk & Cantina"
+}
+```
+
+### Response Structure
+
+All responses with an HTTP status of 200 will consist of a JSON object with a top-level "data" node containing the following elements:
+
+| Field    | Description |
+|----------|-------------|
+| results  | A JSON array of one or more objects containing geography data. If the request returns no results, the array will be empty. |
+| expiration_date | A date string in YYYY-MM-DD format representing the last day that this data may be used. Per our contract with Google, any data cached by users of the this service must be discarded by this date.
+
+&nbsp;
+
+Each element of the returned **results** array will be formatted as follows:
+
+|   |   |
+|---|---|
+| admin_areas | A JSON array of up to 5 admin_area objects. See below for details on the structure of these objects. **This property is always present, but may be empty** if no admin areas were found. |
+| latitude | A double value specifying the location's latitude (in degrees) within the range [-90, +90]. **This property is always present.** |
+| longitude | A double value specifying the location's longitude (in degrees) within the range [-180, +180]. **This property is always present.** |
+| country | The country or region name of an address. **This property may be absent** if the location was not associated with a country. Some locations for which this occurs are "Europe", "Pacific Ocean", and "Gaza Strip" (a contested territory). |
+| locality | The populated place for the address. This typically refers to a city, but may refer to a suburb or a neighborhood in certain countries. **This property may be absent**. |
+| postal_code | The post code, postal code, or ZIP code of an address. **This property may be absent**. |
+| country_code | The two-letter ISO country code. **This property may be absent**. See "Country" for examples of when this might occur. |
+| formatted_address| A string containing the human-readable address of this location. Often this address is equivalent to the "postal address," which sometimes differs from country to country. (Note that some countries, such as the United Kingdom, do not allow distribution of true postal addresses due to licensing restrictions.) **This property may be absent**. |
+| location_type | The classification of the geographic entity returned. Possible values (in descending order of specificity) are STREET_ADDRESS, SUBLOCALITY, POSTAL_CODE, LOCALITY, ADMIN_AREA_2, ADMIN_AREA_1, STREET, COUNTRY, and UNKNOWN. The returned location type will reflect the most specific field present, based on the previously described order. If none of these fields are present (such as for the examples for which the Country field is absent), then the returned location type will be Unknown. **This property is always present and non-empty.** |
```

Comment:
this service shouldn't ever return the larger location_types, right? like, is it the case that this service should actually always return a STREET_ADDRESS location? if so, perhaps we don't need this field, or perhaps we should add that information to the docs...